internal/host/validator.go

```diff
@@ -765,7 +762,8 @@ func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs
 				if _, ok := failedHostIPs[l3.RemoteIPAddress]; !ok {
 					hostname, role, err := GetHostnameAndRoleByIP(l3.RemoteIPAddress, hosts)
 					if err != nil {
-						return nil, err
+						v.log.Warnf("packet loss host validation: %s", err)
```

Comment:
maybe change to something more indicative like ``host experiencing packet loss:`` or ``host validation for packet hermeticity failed with message:``
just a nit

Reply:
sounds good :smile